pkg/pubsub/testdata/config/pubsub.yml

```diff
@@ -0,0 +1,28 @@
+common: &common
+  kafka:
+
+test: &test
+  <<: *common
+  kafka:
+    # Set by APP_PUBSUB_KAFKA_BROKER_URLS env variable
+    broker_urls:
+      - "localhost:9092"
+    # Set by APP_PUBSUB_KAFKA_CLIENT_ID env variable
+    client_id: "test-app"
+    # Set by APP_PUBSUB_KAFKA_CERT_PEM env variable
+    cert_pem: "pem string"
+    # Set by APP_PUBSUB_KAFKA_CERT_PEM_KEY env variable
+    cert_pem_key: "pem key"
+    # Set by APP_PUBSUB_KAFKA_MAX_ATTEMPTS env variable
+    security_protocol: "ssl"
+    enable_ssl_verfification: true
```

Comment:
it's a typo here in the word `verification`

Reply:
Also, can we rename this variable to `ssl_enabled`? In that case, we would have a consistent naming format for the boolean variables - `ssl_enabled`, `publisher_enabled`, `subscriber_enabled`.

Reply:
@terranisu fixed both remarks 👍